projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-suggestions-from-openai/index.js

```diff
@@ -189,7 +189,7 @@ const useSuggestionsFromOpenAI = ( {
 				postContentAbove: getPartialContentToBlock( clientId ),
 				currentPostTitle,
 				options,
-				userPrompt,
+				userPrompt: options?.userPrompt || userPrompt,
```

Comment:
so, following the above suggestion, here we'd give priority to a `presetPrompt` and then fallback to `userPrompt`